backend/src/apiserver/resource/resource_manager.go

```diff
@@ -245,7 +245,7 @@ func (r *ResourceManager) UpdatePipelineDefaultVersion(pipelineId string, versio
 	return r.pipelineStore.UpdatePipelineDefaultVersion(pipelineId, versionId)
 }
 
-func (r *ResourceManager) CreatePipeline(name string, description string, pipelineFile []byte) (*model.Pipeline, error) {
+func (r *ResourceManager) CreatePipeline(name string, description string, namespace string, pipelineFile []byte) (*model.Pipeline, error) {
```

Comment:
nit: `name string, description string, namespace string` -> `name, description, namespace string`
Also, it makes more sense to me to have the name and namespace arguments first (as they are about metadata), but we can also leave it as is.